IOMC/Input/src/MCFileSource.cc

```diff
@@ -32,7 +33,8 @@ MCFileSource::MCFileSource(const ParameterSet & pset, InputSourceDescription con
   }  
   
   reader_->initialize(fileName);  
-  produces<HepMCProduct>();
+  produces<HepMCProduct>("generator");
```

Comment:
I think we still need this later on:

```
process.g4SimHits.Generator.HepMCProductLabel = 'source:generator'
process.genParticles.src = 'source:generator'
```
(instead of just `source`, so it does not make too much difference)

Or is it possible to label the source module as `process.generator`?